src/components/BackOffice/CloseWorkOrders/index.js

```diff
@@ -88,7 +88,44 @@ const CloseWorkOrders = () => {
     return newErrors
   }
 
-  const handleSubmit = (event) => {
+  const renderConfirmationModal = () => {
+    if (showDialog) {
+      return (
+        <Dialog
+          title="Are you sure?"
+          isOpen={showDialog}
+          onDismiss={() => setShowDialog(false)}
+          data-test="confirmation-modal"
+        >
+          <h3 className="lbh-heading-h3">
+            Are you sure you want to change the status of these Work Orders to "
+            {selectedOption == 'CloseToBase' ? 'Completed' : 'Cancelled'}"?
+          </h3>
```

Comment:
The wording needs to be more direct. I don't think this is enough to scare people into being 100% sure they're doing the right thing.

I suggest something more like "Permanently Cancel WorkOrders", or "Complete WorkOrders". Also, the confirmation button should say something like 'Delete workOrders'. Confirm is too vague.

This article is pretty helpful:
https://uxdesign.cc/are-you-sure-you-want-to-do-this-microcopy-for-confirmation-dialogues-1d94a0f73ac6

Reply:
Also, could you add this for the two other forms